v03_pipeline/lib/reference_datasets/exac.py

```diff
@@ -1,10 +1,11 @@
 import hail as hl
 
+from v03_pipeline.lib.misc.nested_field import parse_nested_field
```

Comment:
I don't see this file/function anywhere?

Reply:
Here it is! 

https://github.com/broadinstitute/seqr-loading-pipelines/blob/main/v03_pipeline/lib/misc/nested_field.py

it doesn't need to be moved into the `reference_datasets` folder because it's used when importing normal callsets too.

Reply:
🤦